treeherder/webapp/api/performance_data.py

```diff
@@ -46,12 +46,18 @@ def list(self, request, project):
                 parent_signature__in=parent_signatures)
 
         if not int(request.query_params.get('subtests', True)):
-            signature_data = signature_data.filter(parent_signature__isnull=True)
+            try:
+                signature_data = signature_data.filter(parent_signature__isnull=True)
+            except ValueError:
+                return "No available subtest provided", HTTP_400_BAD_REQUEST
```

Comment:
I don't think we need this, I don't see how we could get a ValueError exception here.